### ISSUE TITLE
error: pathspec 'commit' did not match any file(s) known to git git exited with error code 1

### ISSUE THREAD

**Issue by michael-odonovan** [commitizen/cz-cli#933]
Installed in the normal way, as I have done previously. Now getting this error for some reason?:
```
error: pathspec 'commit' did not match any file(s) known to git
git exited with error code 1
```
From reading, it seems this is an error associated with typos in one line git commits, but cz seems to be working as before but throwing this error once I finish the questionaire. 
Your help much appreciated.

**Comment by Zhengqbbb** [comment]
Try use `cz` command

**Comment by riancintiyo** [comment]
Try use `git cz` instead. It works for me.

**Comment by pedronastasi** [comment]
I was also getting this issue . I'm using Yarn 3. `git cz`  helped me

**Comment by pedronastasi** [comment]
how about you? @michael-odonovan . Are you using Yarn 3?

**Comment by michael-odonovan** [comment]
Sorry guys I decided not to use this anymore. I had a new job and didn't really need another thing in my workflow to troubleshoot. Having said that, I do think it is a brilliant little program.

**Comment by sepsky** [comment]
Hi I'm having the same issue, but I'm using zsh. I read commitizen doesn't really work with zsh, is that correct still? Thanks :)

**Comment by grantwforsythe** [comment]
I am running into the same issue using oh-my-zsh.Neither `cz init` or `git cz init` work for me

**Comment by taragurung** [comment]
Anyone found the fix for it?

**Comment by bruteforks** [comment]
ran into the same...

**Comment by makmn1** [comment]
I also ran into this issue and here's what I learned. There are two CLI tools to work with Commitizen:
1. [commitizen-tools/commitizen](https://github.com/commitizen-tools/commitizen)
2. [commitizen/cz-cli](https://github.com/commitizen/cz-cli) (the repo you're in now)

The first tool is only available through PyPI or the Brew package manager. If you install that tool, you can run `cz commit` and all the other commands that you might have found on [their docs.](https://commitizen-tools.github.io/commitizen/commands/commit/)

The second tool (this one) is available on npm, but based on my testing and the docs, it _does not_ support any of the commands from the first tool. It only supports the `cz` command (and its alias `git cz`). This result is similar to running the `cz commit` command from the first tool. If you try to pass any argument to `cz`, it will error out with:

```
error: pathspec 'whatever-you-tried-to-pass' did not match any file(s) known to git
git exited with error code 1
```

It would be nice if the README added a note on this to prevent confusion, but it seems like this tool is limited compared with the one available on PyPI / Brew.

TL;DR: Use `cz` or `git cz` without any arguments. If you want the commands from [this doc,](https://commitizen-tools.github.io/commitizen/commands/commit/) you need to download the version from PyPI / Brew.

**Comment by Zhengqbbb** [comment]
@makmn1 Try it https://cz-git.qbb.sh/cli/

**Comment by makmn1** [comment]
Thanks for sharing, I just tried it and it looks like a good alternative with the type / scope filtering and commit previews!

**Comment by quyettranvu** [comment]
@makmn1 Thanks for sharing, I can commit with only cz (or git cz), and I decided to install that cli tools from pip package, then I can run commands like in docs using cz and parameters following.
